README.rst

```diff
@@ -25,14 +25,47 @@ Connexion
    :target: https://github.com/zalando/connexion/blob/master/LICENSE
    :alt: License
 
-Connexion is a framework on top of Flask_ to automagically handle your REST API requests
-based on `OpenAPI 2.0 Specification`_ (formerly known as Swagger Spec) files
-in YAML.
+Connexion is a Flask_ application/framework that automagically handles your REST API requests based on `OpenAPI 2.0 Specification`_ (formerly known as Swagger Spec) files in YAML. Written in Python, Connexion enables you to write your Swagger specification, then maps the specification to Python functions. This is what makes it unique from other tools. We built Connexion this way in order to:
```

Comment:
I would just say it is a "framework" because it cannot handle automagically your REST API requests without you actually writing some code.